Framework/Geometry/inc/MantidGeometry/Objects/MeshObject.h

```diff
@@ -202,7 +203,8 @@ class MANTID_GEOMETRY_DLL MeshObject : public IObject {
   boost::shared_ptr<vtkGeometryCacheWriter> m_vtk_cache_writer;
 
   /// Contents
-  std::vector<size_t> m_triangles;
+  /// Triangles are specified by indices into a list of vertices.
+  std::vector<uint16_t> m_triangles;
```

Comment:
We discussed on the PR with @martyngigg that this should be `uint32_t`. `unit16_t` is probably sufficient but we agreed on `uint32_t`

As a side point, as that would give >65K addresses for vertexes. Assuming a `V3D` is is 3x8bytes, in the worst case this would give a `MeshObject` the potential size of 1.5GB.  Using `uint32_t`, we get a max size of 96GB in the worst case, per `MeshObject`

Reply:
@OwenArnold Now you phrase it that way maybe `uint16_t` is sufficient. We probably don't want a 96Gb mesh! At least more will fit in the cache if they are smaller.